addons/repair/models/repair.py

```diff
@@ -303,8 +303,9 @@ def _create_invoices(self, group=False):
             currency = repair.pricelist_id.currency_id
             # Fallback on the user company as the 'company_id' is not required.
             company = repair.company_id or self.env.user.company_id
+            # VFE FIXME why user company and not environment company as fallback ?
```

Comment:
@sle-odoo When creating invoices from repair, when the repair has no company_id, the user company is used as fallback and not the one from the environment.  Is it done on purpose (or a remains not seen by yti during his multi company changes) ?